OpenRA.Mods.Common/Activities/Move/Move.cs

```diff
@@ -52,20 +49,14 @@ public class Move : Activity
 		bool evaluateNearestMovableCell;
 
 		// Scriptable move order
-		// Ignores lane bias and nearby units
```

Comment:
Likewise here. I vaguely remember trying to remove this special case in the past to streamline the logic, and having to abandon the idea because of regressions.

Reply:
Restored the option per other comment.